generators/emails/templates/example_email.rb

```diff
@@ -3,6 +3,6 @@ class TestEmail < Howitzer::Email
   subject 'Test email'
 
   def addressed_to?(new_user)
-    /Hi #{new_user}/ === plain_text_body
+    plain_text_body.match?(/Hi #{new_user}/)
```

Comment:
We have to support Ruby 2.2.2 https://github.com/strongqa/howitzer#requirements
There is no such method in https://ruby-doc.org/core-2.2.2/String.html